### PR TITLE
Add parent description for Eljin asset

### PR DESCRIPTION
**Pull request by Pantani**
Issue [310](https://github.com/trustwallet/blockatlas/issues/310)

### CODE REVIEW

File: platform/ethereum/api.go [hunk 1]
```diff
@@ -216,11 +216,15 @@ func NormalizeCollectionPage(collections []Collection, coinIndex uint, owner str
 }
 
 func NormalizeCollection(c Collection, coinIndex uint, owner string) blockatlas.Collection {
+	description := c.Contracts[0].Description
+	if len(description) == 0 {
+		description = c.Description
+	}
```
Comment (vikmeup) [inline platform/ethereum/api.go]: do you have a test for this by any chance? would be good to see an example

Comment (Pantani) [reply platform/ethereum/api.go]: http://localhost:8420/v2/ethereum/collections/0x0875BCab22dE3d02402bc38aEe4104e1239374a7

Comment (Pantani) [reply platform/ethereum/api.go]:
```
[
  {
    "name": "CryptoKitties",
    "symbol": "CKITTY",
    "image_url": "https://storage.opensea.io/0x06012c8cf97bead5deae237070f9587f8e7a266d-featured-1556588705.png",
    "description": "CryptoKitties is a game centered around breedable, collectible, and oh-so-adorable creatures we call CryptoKitties! Each cat is one-of-a-kind and 100% owned by you; it cannot be replicated, taken away, or destroyed.",
    "external_link": "https://www.cryptokitties.co/",
    "total": 2,
    "category_address": "0x06012c8cf97bead5deae237070f9587f8e7a266d",
    "address": "0x0875BCab22dE3d02402bc38aEe4104e1239374a7",
    "nft_version": "1.0",
    "coin": 60,
    "type": "ERC721"
  },
  {
    "name": "Known Origin",
    "symbol": "KODA",
    "image_url": "https://storage.opensea.io/0xfbeef911dc5821886e1dda71586d90ed28174b7d-featured-1556588770.png",
    "description": "KnownOrigin brings together blockchain technology and digital artwork though the power of ERC721 backed digital arts assets.",
    "external_link": "https://knownorigin.io",
    "total": 1,
    "category_address": "0xdde2d979e8d39bb8416eafcfc1758f3cab2c9c72",
    "address": "0x0875BCab22dE3d02402bc38aEe4104e1239374a7",
    "nft_version": "3.0",
    "coin": 60,
    "type": "ERC721"
  },
  {
    "name": "Enjin",
    "symbol": "",
    "image_url": "https://storage.opensea.io/0x8562c38485b1e8ccd82e44f89823da76c98eb0ab-featured-1556588805.png",
    "description": "Enjin assets are unique digital ERC1155 assets used in a variety of games in the Enjin multiverse.",
    "external_link": "https://enj1155.com",
    "total": 1,
    "category_address": "0xfaafdc07907ff5120a76b34b731b278c38d6043c",
    "address": "0x0875BCab22dE3d02402bc38aEe4104e1239374a7",
    "nft_version": "",
    "coin": 60,
    "type": "ERC1155"
  }
]```

Comment (vikmeup) [reply platform/ethereum/api.go]: is this test inside unit testing? it would be great to have it there

Comment (vikmeup) [reply platform/ethereum/api.go]: Should we set default "nft_version" ?

Comment (Pantani) [reply platform/ethereum/api.go]: ohh, unit test, yes! I can do it. I fixed this code because I got some panic cases when I run the integrations tests.

Comment (Pantani) [reply platform/ethereum/api.go]: About `nft_version`, I don't know, @ArtemGoryunov said,  it's ok for 1155 tokens. If the app doesn't break, we can keep with this parameter empty

Comment (Pantani) [reply platform/ethereum/api.go]: uniter test, done!

**Pull request merged**